netci.groovy

```diff
@@ -28,30 +25,35 @@ platformList.each { platform ->
     def buildCommand = '';
     def osForGHTrigger = os
     def version = "latest-or-auto"
+    def dockerRepository = "microsoft/dotnet-buildtools-prereqs"
+    def dockerContainer = ''
+    def dockerWorkingDirectory = "/src/core-setup"
+    def dockerCommand = ''
```

Comment:
I think this variable can be moved to where it is used.  It is only used in one spot - no need for it to be at this scope.